ocp/jenkins.sh

```diff
@@ -39,6 +47,9 @@ oc set env dc/jenkins INSTALL_PLUGINS="${JENKINS_PLUGINS}"
 oc set env dc/jenkins CASC_JENKINS_CONFIG="/var/lib/jenkins/init.groovy.d/casc_jenkins.yaml"
 oc set volumes dc/jenkins --add --configmap-name=jenkins-configuration --mount-path='/var/lib/jenkins/init.groovy.d/' --name "jenkins-config"
 oc set volumes dc/jenkins --add --configmap-name=jenkins-configuration --mount-path='/var/lib/jenkins/.config/yamllint' --name "yamllint-config"
+oc set volumes dc/jenkins --add --secret-name=jenkins-ci-github-key --mount-path='/var/lib/jenkins/.ssh/id_rsa' --name "jenkins-ci-github-key" --read-only=true
```

Comment:
doesn't this conflict with the credentials id defined in casc_jenkins.yaml?

Reply:
sshagent doesn't work, so we'll have to stick to this